src/nerve/frontends/cli/repl/adapters.py

```
@@ -656,3 +656,138 @@ async def execute_python(self, code: str, namespace: dict[str, Any]) -> tuple[st
+        """
+        from nerve.server.protocols import Command, CommandType
+
+        result = await self.client.send_command(
+            Command(type=CommandType.LIST_WORKFLOWS, params=self._add_session_id({}))
+        )
+        if result.success:
+            return cast(list[dict[str, Any]], (result.data or {}).get("workflows", []))
+        return []
+
+    async def execute_workflow(
+        self, workflow_id: str, input: Any = None, wait: bool = False
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `input`

(ANN401)